farcy/__init__.py

```diff
@@ -49,6 +49,52 @@
 PR_ISSUE_COMMENT_FORMAT = '_{0}_ {{0}}'.format(MD_VERSION_STR)
 COMMIT_STATUS_FORMAT = '{0} {{0}}'.format(VERSION_STR)
 
+FARCY_COMMENT_START = '_{}_'.format(MD_VERSION_STR)
+
+
+def extract_issues(text):
```

Comment:
I moved it all into a helpers file. I also created a const file in the process because `helpers` and `__init__` both depended on constants. I'm open for any suggestion to change it.